lib/pages/bootstrap/tom_bootstrap_dialog.dart

```diff
@@ -57,19 +56,17 @@ class TomBootstrapDialogState extends State<TomBootstrapDialog>
   @override
   void initState() {
     super.initState();
-    bootstrap =
-        widget.client.encryption!.bootstrap(onUpdate: (_) => setState(() {}));
-    _createBootstrap();
+    _initData();
   }
 
-  void _createBootstrap() async {
+  void _initData() async {
     WidgetsBinding.instance.addPostFrameCallback((_) async {
       await _loadingData();
     });
   }
 
   Future<void> setupAdditionalDioCacheOption(String userId) async {
-    Logs().d('TomBootstrapDialog::setupAdditionalDioCacheOption: $userId');
+    Logs().i('TomBootstrapDialog::setupAdditionalDioCacheOption: $userId');
```

Comment:
IMO those logs should get back to `Logs().d`

Reply:
don't worry, its only appear when you debug